solana/gateway-ts/docs/classes/Active.html

```diff
@@ -111,7 +111,7 @@ <h3>constructor</h3>
 							<aside class="tsd-sources">
 								<p>Inherited from Assignable.constructor</p>
 								<ul>
-									<li>Defined in <a href="https://github.com/identity-com/on-chain-identity-gateway/blob/83df0519/solana/gateway-ts/src/lib/solanaBorsh.ts#L7">lib/solanaBorsh.ts:7</a></li>
+									<li>Defined in <a href="https://github.com/identity-com/on-chain-identity-gateway/blob/17c33ed6/solana/gateway-ts/src/lib/solanaBorsh.ts#L7">lib/solanaBorsh.ts:7</a></li>
```

Comment:
Are all the change to the docs html generated?

Reply:
Yes via oclif